Applications/Spire/Include/Spire/OrderImbalanceIndicator/FilteredOrderImbalanceIndicatorModel.hpp

```diff
@@ -69,9 +68,12 @@ namespace Spire {
   //! list.
   /*
     \param market_list The list of markets to preserve after filtering.
+    \param market_database Market database used for converting display names
+                           to market codes.
   */
   FilteredOrderImbalanceIndicatorModel::Filter make_market_list_filter(
-    const std::set<std::string>& market_list);
+    const std::set<std::string>& market_list,
+    const Nexus::MarketDatabase& market_database);
```

Comment:
When I make a suggestion it's not feasible for me to go over the entire pull request and point out every line that the suggestion applies to. Review your pull request to make sure you are properly applying the suggestions to all instances instead of depending on me to point out every single instance of it. Otherwise these code reviews will end up taking much longer than is needed.